### ISSUE TITLE
Network manager remove old stuff when you switch net or connect to vpn (landrush)

### ISSUE THREAD

**Issue by amitkrout** [projectatomic/adb-atomic-developer-bundle#518]
Openshift web console does not reconnect when vpn is reconnect or when you switch net.

```
$ ping foo.cdk
ping: unknown host foo.cdk
$ ping openshift.cdk
ping: unknown host openshift.cdk
$
```

After adding `nameserver 127.0.0.1` to the file `/etc/resolv.conf` resolves the problem.

**Comment by praveenkumar** [comment]
This is something we should add as troubleshooting doc when landrush is used. Bug is only signify for Linux (Redhat/CentOS/Fedora) because `/etc/resolve.conf` is under NetworkManager command so whenever vpn connected or network switch happen then this file is overwritten and loose the landrush change which is simple one line it add at top of that file.

```
# Added by landrush, a vagrant plugin 
nameserver 127.0.0.1 
```

**Comment by bexelbie** [comment]
This is because landrush doesn't properly register itself with networkmanager on enterprise linux

**Comment by bexelbie** [comment]
https://github.com/vagrant-landrush/landrush/issues/260

**Comment by Preeticp** [comment]
Closed via #559

**Comment by Preeticp** [comment]
This is documented the troubleshooting guide at https://github.com/projectatomic/adb-atomic-developer-bundle/blob/master/docs/troubleshooting.adoc
